flink-addons/flink-language-binding/src/main/python/org/apache/flink/languagebinding/api/python/dill/__diff.py

```diff
@@ -0,0 +1,247 @@
+################################################################################
```

Comment:
We need to find another solution here with the licenses. I think we can not just re-distribute this file with our license.